### ISSUE TITLE
FNX-4592 ⁃ [Bug] UI Test Failure: validateBaselinePing in org.mozilla.fenix.glean.BaselinePingTest

### ISSUE THREAD

**Issue by AaronMT** [mozilla-mobile/fenix#10652]
https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6672861841223743063/executions/bs.55755e76fba6649/testcases/1/test-cases

```Kotlin
kotlin.KotlinNullPointerException
at org.mozilla.fenix.glean.BaselinePingTest.validateBaselinePing(BaselinePingTest.kt:119)
```

https://github.com/mozilla-mobile/fenix/blob/824dedb19588a9052b03ad162155c62ecd08e316/app/src/androidTest/java/org/mozilla/fenix/glean/BaselinePingTest.kt#L119

**Comment by AaronMT** [comment]
(@Dexterp37 have you had a chance to take a look at this?)

**Comment by AaronMT** [comment]
Disabling for now as it's a consistent failure.

**Comment by Dexterp37** [comment]
Closed by #11254
